src/partitioned-dbs/index.rst

```diff
@@ -130,6 +136,33 @@ the query across all ``q`` shards in the database. This mean that you do
 not have to wait for all ``q`` shards to respond, which is both
 efficient and faster.
 
+In other words, you can use partitions to help pre-process views. As another
+example, you may be recording traffic to various websites, and 95% of your
+queries are based around 'what happened in relation to website example.com'.
+By partitioning your database with the key ``example.com:uuid``, and using
+the ``/dbname/_partition/example.com`` query, you are significantly simplifying
+the work the query engine has to do.
```

Comment:
Compare and contrast this argument with that we make for Mango partial indexes - where we artificially limit the documents in a Mango index based on an initial condition. How does this differ from our suggestion there? The text is unclear to the prospective user.